api/database/migrations/2023_02_01_194314_connect_teams_pools.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+use App\Models\Team;
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // this Team is seeded in TeamSeeder.php, it is essential this team exists BEFORE migrating so the updateOrCreate is run to make sure
+        Team::updateOrCreate(
+            [
+                'name' => 'digital-community-management',
+            ],
+            [
+                'display_name' => [
+                    'en' => 'Digital Community Management',
+                    'fr' => 'Gestion de la collectivité numérique',
+                ],
+            ],
+        );
+
+        Schema::table('pools', function (Blueprint $table) {
+            $table->uuid('team_id')->references('id')->on('teams')->nullable()->default(null);
+        });
+
+        $dcmIdQuery = DB::select('SELECT id FROM teams WHERE name = :name', ['name' => 'digital-community-management']);
+        $dcmId = $dcmIdQuery[0]->id;
+
+        DB::statement(
+            <<<SQL
+                UPDATE pools
+                    SET team_id = :dcmId
+            SQL, [
+                'dcmId' => $dcmId,
+            ]);
```

Comment:
I think we can use eqloquent here to:

```php
  Pool::all()
    ->team()
    ->save($dcmId);
```

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/5561/commits/962c19e480eb1cff4a3a9b6c40abcff53bcd6188

Reply:
it didn't like that method, so I went back to the tried and true 🤷 
https://github.com/GCTC-NTGC/gc-digital-talent/pull/5561/commits/139c3d633dd8db903c3dea6cd1e6710cda7e8b46

Reply:
Sorry, `all()` returns a collection so you would need to loop through them 😅 

```php
Pool::all()
  ->each(function ($pool) use ($dcmId) {
    $pool->team()->save($dcmId);
  });
```

Another option would to just set it as the default 🤔 Since we are removing the default later on.

```php
$dcmId = Team::where('name', 'digital-community-management')
    ->first()->id;

Schema::table('pools', function (Blueprint $table) use ($dcmId) {
    $table->uuid('team_id')->references('id')->on('teams')->nullable()->default($dcmId);
});
```

I just don't think it is good practice to rely on SQL. Eloquent is a safer way of writing these statements and has a lot of stuff happening in the background related to compatibility with postrgres and other SQL servers (in the event we decide to replace it in the future)

Reply:
I see, well, it is good to have a standard to use
wow, the set default directly was a way better idea thanks, https://github.com/GCTC-NTGC/gc-digital-talent/pull/5561/commits/8559089a2213c9bf70000b5702229ffe91261ffa

Reply:
actually nevermind lol
setting it as a default that require a migration to reverse down the line seems like it'd create an annoyance for someone, bugs possibly

lets not https://github.com/GCTC-NTGC/gc-digital-talent/pull/5561/commits/c22dc06ccf82edc04d180c5681642b43174a1073

Reply:
Looks like Eloquent supports mass updates: https://laravel.com/docs/9.x/eloquent#mass-updates